nemo_curator/modules/semantic_dedup/embeddings.py

```diff
@@ -217,6 +217,7 @@ def __call__(self, dataset: DocumentDataset) -> DocumentDataset:
                 )
             )
         else:
+            embedding_ddf = self.create_embeddings(dataset.df, self.input_column)
```

Comment:
This is the error that would pop up when doing `write_to_disk`: https://github.com/rapidsai/cudf/issues/10115.

Reply:
Will be captured by: https://github.com/NVIDIA/NeMo-Curator/issues/505